tempodb/backend/azure/azure.go

```diff
@@ -132,12 +138,13 @@ func (rw *readerWriter) CloseAppend(context.Context, backend.AppendTracker) erro
 }
 
 func (rw *readerWriter) Delete(ctx context.Context, name string, keypath backend.KeyPath, _ bool) error {
-	blobURL, err := GetBlobURL(ctx, rw.cfg, backend.ObjectFileName(keypath, name))
+	blobClient, err := getBlobClient(ctx, rw.cfg, backend.ObjectFileName(keypath, name))
 	if err != nil {
-		return errors.Wrapf(err, "cannot get Azure blob URL, name: %s", backend.ObjectFileName(keypath, name))
+		return errors.Wrapf(err, "cannot get Azure blob client, name: %s", backend.ObjectFileName(keypath, name))
```

Comment:
Are we OK to rename errors? I guess this could _technically_ be considered a breaking change? As a mostly academic argument, one might imagine that a Tempo consumer has an alert rule set up to listen for a log statement like `cannot get Azure blob URL`. In this case, this change would break that rule. However, for all intents and purposes, I'd consider this change OK. This also applies to other errors that have been renamed.

Thoughts?

Reply:
I'm fine with changing the text of a log line and not list it as a breaking change.